GridLockDemoApp/GridLockDemoApp/AppDelegate.swift

```diff
@@ -12,12 +12,16 @@ import GridLock
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-    var window: UIWindow?
-    var gridLock: GridLock?
-
+    var gridLock = GridLock()
+    var window: UIWindow? {
```

Comment:
You can omit the empty setter to define a read only computed property e.g.
```
var window: UIWindow? {
    return gridLock.window
}
```

Reply:
I can't do that here because the `UIApplicationDelegate` protocol demands a setter!

`optional public var window: UIWindow? { get set }`

If I remove the get and sets, there is a warning that we are `nearly matching the protocol` lol